client/src/pages/ProfilePage.tsx

```diff
@@ -103,46 +84,22 @@ export default function ProfilePage() {
       <div className="profile-base-wrapper p-4 border-y-solid border-y-gray-400 border-y-1px">
         <div className="flex flex-row gap-4 pb-4">
           {/*TO DO: have to add that the user image is the src if there is */}
-          {user?.image ? (
-            <Image src={user?.image} alt="Profile Image" className="rounded-full h-24 w-24" />
-          ) : (
-            <Icon
-              name={ICON_NAMES.profile_placeholder}
-              height={91}
-              width={91}
-              viewBox="0 0 91 91"
-              className="rounded-full"
-            />
-          )}
+          {user?.image ? <Image src={user?.image} alt="Profile Image" className="rounded-full h-24 w-24" /> : <Icon name={ICON_NAMES.profile_placeholder} height={91} width={91} viewBox="0 0 91 91" className="rounded-full" />}
 
           <div className="flex flex-col">
             <div className="flex flex-row gap-4 items-center">
               <TextHeadline variant="h1" size="sm">
                 {fullName}
               </TextHeadline>
-              {user?.isAvailable && (
-                <ProfileBadge ProfileBadgeLabel="Seeking" ProfileBadgeSize="sm" />
-              )}
+              {user?.isAvailable && <ProfileBadge ProfileBadgeLabel="Seeking" ProfileBadgeSize="sm" />}
             </div>
-            <TextBody>{user?.createdAt?.toDateString()}</TextBody>
-            <TextBody>{user?.lastLoggedIn?.toDateString()}</TextBody>
+            <TextBody>{formatDate(user.createdAt)}</TextBody>
+            <TextBody>{formatDate(user.lastLoggedIn)}</TextBody>
           </div>
         </div>
         <div className="flex flex-row gap-4">
-          <Button
-            buttonVariant="secondary"
-            to="/profile/$profileId/edit"
-            params={{ profileId: userId }}
-            iconPosition="none"
-            buttonLabel="Edit Profile"
-            className="no-underline"
-          ></Button>
-          <Button
-            buttonVariant="secondary"
-            onClick={() => console.log("Settings")}
-            iconPosition="none"
-            buttonLabel="Settings"
-          ></Button>
+          <Button buttonVariant="secondary" to="/profile/$profileId/edit" params={{ profileId: userId ? userId : "" }} iconPosition="none" buttonLabel="Edit Profile" className="no-underline"></Button>
```

Comment:
can't it cause problems if in this phase, where we are going from the userprofile page to editing it, the user id can be undefined?